VeterinaryElena/Program.cs

```diff
@@ -10,6 +10,131 @@ class Program
+                ",\n R - put new animal in clinic, \n L - show list of animals,\n D - discharge animal,\n Q - quit");
+           
+
+            action = Console.ReadLine();
+
+            if (action == "R" || action == "r")
+            {
+                Console.WriteLine("Enter kind of animal(cat, dog, humster or fish)");
+                string kind = Console.ReadLine();
+                Console.WriteLine("Enter owner of animal");
+                string owner = Console.ReadLine();
+                Console.WriteLine("Enter name of animal");
+                string nameAnimal = Console.ReadLine();
+
+                int id = 0;
+                id=newAnimal.isConteins(nameAnimal, owner, kind);
+                if (id != 0)
+                {
+                    newAnimal.Readmission(id);
+                }
+                else
+                {
+                    Console.WriteLine("Enter year of birthday of animal");
+                    int year;//= Console.ReadLine();
+                    while (!Int32.TryParse(Console.ReadLine(), out year))
+                    { Console.WriteLine("You enter not correct number. Please, enter number"); }
+                    int thisYear = Convert.ToInt32(DateTime.Now.Year);
+
+                    Console.WriteLine("Select your doctor:\n 1- surgeon\n 2- traumatologist");
+                    int doctorID;
+                    while (!Int32.TryParse(Console.ReadLine(), out doctorID) || doctorID < 1 && doctorID > 2)
+                    { Console.WriteLine("You enter not correct number. Please, enter number"); }
+
+
+                    if (kind == "cat")
+                    {
+
+                        HomeCat newCat = new HomeCat(nameAnimal, doctorID, thisYear - year, false, owner);
+                        if (thisYear - year > newCat.AgeAverage)
+                        { Console.WriteLine("Your cat is very old. Is data true?"); }
+                        newAnimal.animals.Add(newCat);
+                    }
+
+                    if (kind == "dog")
+                    {
+
+                        HomeDog newDog = new HomeDog(nameAnimal, doctorID, thisYear - year, false, owner);
+                        if (thisYear - year > newDog.AgeAverage)
+                        { Console.WriteLine("Your dog is very old. Is data true?"); }
+                        newAnimal.animals.Add(newDog);
+                    }
+
+                    if (kind == "hamster")
+                    {
+
+                        HomeHamster newHumster = new HomeHamster(nameAnimal, doctorID, thisYear - year, false, owner);
+                        if (thisYear - year > newHumster.AgeAverage)
+                        { Console.WriteLine("Your humster is very old. Is data true?"); }
+                        newAnimal.animals.Add(newHumster);
+                    }
+
+                    if (kind == "fish")
+                    {
+                        bool predatory=false;
```

Comment:
I believe this "predatory" flag should belong to common Animal class and be initialized through c'tor